src/adapter_covid19/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Hey Bryn,

This is slightly complicated, but this module can be Apache licensed as long as it doesn't use or interface with any of the GPL code that already exists in the repository (which is the case). The rest of the repository *can* use this module, as code licensed under Apache can be used by the GPL.

https://www.apache.org/licenses/GPL-compatibility.html

We talked with @sbalian and others about this, including getting a legal opinion, please let me know if this doesn't make sense or if it would be helpful to loop others in to explain.

Thanks

Reply:
Yep! It was just the first commit that we needed to be under Apache.